assets/scss/_footer.scss

```diff
@@ -1,32 +1,24 @@
 .footer{
-    background-image: url("../images/footerbg.svg");
-    background-size: cover;
-    background-repeat: no-repeat;
+    background-image: url("../images/footerbg.svg") cover no-repeat;
```

Comment:
only background can take all the properties, so please use it instead `background-image`

Reply:
done